src/functionalTest/groovy/org/betterdevxp/testkit/GradleRunnerSupport.groovy

```diff
@@ -22,10 +22,16 @@ trait GradleRunnerSupport {
     }
 
     BuildResult run(String... args) {
+        if ((args as List).contains("-s") == false) {
+            args = args + "-s"
```

Comment:
can this argument be a constant?

Reply:
other option is provide method for checking for this argument as it is repeated in both run methods